src/main/java/mate/academy/util/HibernateUtil.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.util;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+    private static final SessionFactory instance = initSessionFactory();
+    
+    public HibernateUtil() {
```

Comment:
```suggestion
    private HibernateUtil() {
```
The point is to have private constructor, because we have only static methods and we don't want to create HibernateUril instances anywhere

Reply:
done